lib_nbgl/include/nbgl_use_case.h

```diff
@@ -137,10 +137,67 @@ typedef struct {
     uint8_t nbContents;  ///< number of contents
 } nbgl_genericContents_t;
 
+/**
+ * @brief The different types of operation to review
+ *
+ */
+typedef enum {
+    TYPE_OPERATION = 0,
+    TYPE_TRANSACTION,
+    TYPE_MESSAGE,
+} nbgl_operationType_t;
+
+/**
+ * @brief The different types of review status
+ *
+ */
+typedef enum {
+    STATUS_TYPE_OPERATION_SIGNED = 0,
+    STATUS_TYPE_OPERATION_REJECTED,
+    STATUS_TYPE_TRANSACTION_SIGNED,
+    STATUS_TYPE_TRANSACTION_REJECTED,
+    STATUS_TYPE_MESSAGE_SIGNED,
+    STATUS_TYPE_MESSAGE_REJECTED,
+    STATUS_TYPE_ADDRESS_VERIFIED,
+    STATUS_TYPE_ADDRESS_REJECTED,
+} nbgl_reviewStatusType_t;
+
 /**********************
  * GLOBAL PROTOTYPES
  **********************/
 
+void nbgl_useCaseHomeAndSettings(
+    const char                *appName,
+    const nbgl_icon_details_t *appIcon,
+    const char                *tagline,
+    const uint8_t *initSettingPage,  // Set to NULL if settings shouldn't be displayed right away
+    const nbgl_genericContents_t *settingContents,
+    const nbgl_contentInfoList_t *infosList,
+    const char                   *actionText,      // Set to NULL if no additional action
+    nbgl_callback_t               actionCallback,  // Set to NULL if no additional action
+    nbgl_callback_t               quitCallback);
+
+void nbgl_useCaseReview(nbgl_operationType_t             operationType,
+                        const nbgl_layoutTagValueList_t *tagValueList,
+                        const nbgl_icon_details_t       *icon,
+                        const char                      *reviewTitle,
+                        const char *reviewSubTitle, /* Most often this is empty, but sometimes
+                                                       indicates a path / index */
+                        const char           *finishTitle, /* unused on Nano */
+                        nbgl_choiceCallback_t choiceCallback);
```

Comment:
nbgl_useCaseAddressReview already knows the `nbgl_operationType_t`
And it if we succeed or fail, so it can deduce the `nbgl_reviewStatusType_t`

Reply:
By doing so we remove one useless callback, and prevent the wrong way to do it, seen on some apps :
`static void review_choice(bool confirm) {
    // Answer, display a status page and go back to main
    if (confirm) {
        nbgl_useCaseReviewStatus(STATUS_TYPE_ADDRESS_VERIFIED, ui_menu_main);
    } else {
        nbgl_useCaseReviewStatus(STATUS_TYPE_ADDRESS_REJECTED, ui_menu_main);
    }
    validate_pubkey(confirm);
}`

Reply:
This will prevent from removing  the need for the `validate_pubkey` callback in a sync version...
So this won't be that sync...
And Rust app will have to provide a C callback, which is a mess? @agrojean-ledger

Reply:
Yeah, the non compatibility with sync scenario makes me says that I wouldn't do it personally.
Having `validate_pubkey` being calls as a callback, enforce exposing the say pubkey in a global variable and all.
We can always expose both, but I doubt this is the best idea...